### PR TITLE
[TF 2.0 API Docs] tf.data Add pointer to tutorials which work with r2.0

### PR DESCRIPTION
**Pull request by netskink**

### CODE REVIEW

File: tensorflow/python/data/ops/dataset_ops.py [hunk 1]
```diff
@@ -92,6 +92,71 @@ class DatasetV2(tracking_base.Trackable, composite_tensor.CompositeTensor):
   A `Dataset` can be used to represent an input pipeline as a
   collection of elements (nested structures of tensors) and a "logical
   plan" of transformations that act on those elements.
+  
+  
+For example:
+
+    ```python
+    tf.print("tf.keras.model.fit() can take a single tf.data.Dataset for training features and labels. \
```
Comment (alextp) [inline tensorflow/python/data/ops/dataset_ops.py]: Should also mention that if you use tf.keras.layers.Input(..., name=name) the dataset can also return a dictionary with the names of the inputs as keys.

**Review by alextp — CHANGES_REQUESTED** [review]

File: tensorflow/python/data/ops/dataset_ops.py (hunk 1 above)
Comment (netskink) [reply tensorflow/python/data/ops/dataset_ops.py]: I was unaware of that capability!  But I’m excited to try it out.  I appreciate the response and I’ll edit and resubmit.

Comment (netskink) [reply tensorflow/python/data/ops/dataset_ops.py]: I was unaware of that capability!  But I’m excited to try it out.  I appreciate the response and I’ll edit and resubmit.

Comment (alextp) [reply tensorflow/python/data/ops/dataset_ops.py]: Please ping here when you want me to take another look :-)

**Comment by lamberta** [comment]
Links added automatically now. Thanks
